remote/src/components/Routing.jsx

```diff
@@ -1,25 +1,43 @@
 import React from "react";
-import { BrowserRouter, Routes, Route } from "react-router-dom";
+import { Routes, Route, BrowserRouter } from "react-router-dom";
 import NavBar from "./NavBar";
 import PageNotFound from "./pages/PageNotFound";
 import Contact from "./pages/Contact";
 import Home from "./pages/Home";
 import About from "./pages/About";
+import ContactAddress from "./pages/ContactAddress";
+import ContactNumber from "./pages/ContactNumber";
+import ContactMail from "./pages/ContactMail";
+import ContactNumber1 from "./pages/ContactNumber1";
+import ContactNumber2 from "./pages/ContactNumber2";
+import ContactNumber3 from "./pages/ContactNumber3";
+import HomeContact from "./pages/HomeContact";
+import OfficeContact from "./pages/OfficeContact";
 
 export default function Routing() {
   return (
     <div>
-      <BrowserRouter>
-      <h1>Remote...</h1> <br /><br />
-        <NavBar />
-        <Routes>
-          <Route path="/*" element={<PageNotFound />} />
-          <Route path="/" element={<Home />} />
-          <Route path="/remote-home" element={<Home />} />
-          <Route path="/remote-contact" element={<Contact />} />
-          <Route path="/remote-about" element={<About />} />
-        </Routes>
-      </BrowserRouter>
+      <h1>Remote...</h1> <br />
+      <br />
+      <NavBar />
+      <Routes>
+        <Route path="/*" element={<PageNotFound />} />
```

Comment:
Try using only * I believe that could also work fine.

Reply:
Done